ui/components/pagination.js

```diff
@@ -0,0 +1,95 @@
+import { useRouter } from 'next/router'
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/solid'
+
+function getPageNums(selected, count, totalPages) {
```

Comment:
you can move this function in `Pages`  because no one else is using this function.